lib/core/src/lib/card-view/models/card-view-baseitem.model.spec.ts

```diff
@@ -119,5 +119,34 @@ describe('CardViewBaseItemModel', () => {
             expect(isValid).toBe(false);
             expect(itemModel.getValidationErrors('test-against-this')).toEqual([validator1, validator3 ]);
         });
+
+        it('should validate when there is REGEX constraint and requiresMatch is false', () => {
+            const constrainedProperties = {
+                ...properties,
+                value: 'test.',
+                constraints: [{
+                    id: 'constraint-id',
+                    type: 'REGEX',
+                    parameters: { expression: '(.*[\\"\\*\\\\\\>\\<\\?\\/\\:\\|]+.*)|(.*[\\.]?.*[\\.]+$)|(.*[ ]+$)', requiresMatch: false }
+                }]
+            };
+
+            const itemModel = new CarViewCustomItemModel(constrainedProperties);
+            expect(itemModel.isValid(itemModel.value)).toBe(false);
+        });
+
+        it('should validate when there is REGEX constraint and requiresMatch is true', () => {
+            const constrainedProperties = {
```

Comment:
could you please set type to this const?

Reply:
added

Reply:
@jacekpluta I still can't see type here